content/22.vaccines.md

```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
 
 ### Live-Attenuated Viruses
 
-<!--To Do: Not used this time I think?-->
-Although this is the vaccine development strategy with the longest history... (short paragraph explaining not really in play here)
+Live-attenuated vaccines (LAV) or replication-competent vaccines use a weakened living version of a disease-causing virus or a version of a virus that is similar to induce an immune response [@doi:10.1038/s41586-020-2798-3]. 
+The virus can be attenuated by passinging it in a foreign host, until as a consequence of selection pressure the virus loses its efficacy in the original host. 
+Alternatively, selective gene deletion or codon de-optimization can be utilized to attenuate the virus [@doi:10.1038/s41586-020-2798-3]. 
+LAVs are used globally to prevent diseases caused by viruses such as measles, rubella, polio, influenza, varicella zoster, and the yellow fever virus [@doi:10.1016/B978-0-323-35761-6.00002-X].
```

Comment:
Actually all those listed are viruses as is..., I just shortened it to rubella instead of rubella virus, to reduce the length of the sentence. e.g. such as measles virus, rubella virus, polio virus, influenza virus, yellow fever virus, and those caused by Varicella zoster virus.

So should we italicize them all, or leave it as is...

Reply:
Oops! I totally misread it. Let's just leave it how you wrote it and let the copy editors tell us if we're wrong!